### DISCUSSION TITLE
Auto Redeploy

### DISCUSSION THREAD

**Discussion by gardito-git** [cuigh/swirl#56] [Q&A] [answered]
Can a Swirl auto-redeploy in the same tag like latest ?
Usually production environment updated once-twice a week using tag GA-* or latest in docker image, my question can a swirl auto-detect / auto-deploy same tag latest if there is a new image in private Docker Harbor repository ?

**Comment by vanzhiganov** [comment, marked as answer]
I think using the latest tag (or another tags) with continuous updates after deploy is a dirty way that may give some problems in environments.
Another (better?) way is implementation functions for detecting updates on docker hub or specified docker registry.
